modules/appeals_api/spec/support/shared_examples_for_appeal_status_updates.rb

```diff
@@ -71,38 +71,35 @@
 
     context "when status has updated to 'submitted' and claimant or veteran email data present" do
       it 'enqueues the appeal received job' do
-        expect(AppealsApi::AppealReceivedJob.jobs.size).to eq 0
+        expect(AppealsApi::AppealSubmittedJob.jobs.size).to eq 0
```

Comment:
Note that Flipper flags are enabled by default in tests - that's why this class name changes in specs, but nothing else does - this is the only part controlled by the Flipper flag (the two jobs have their own tests)